package.json

```diff
@@ -74,12 +74,12 @@
   },
   "dependencies": {
     "@axelar-network/axelarjs-sdk": "^0.6.18",
-    "@chakra-ui/icons": "^2.0.19",
-    "@chakra-ui/react": "^2.7.0",
-    "@chakra-ui/system": "^2.5.8",
-    "@chakra-ui/tag": "^2.0.2",
-    "@emotion/react": "^11.9.3",
-    "@emotion/styled": "^11.9.3",
+    "@chakra-ui/icons": "^2.1.0",
+    "@chakra-ui/react": "^2.8.0",
+    "@chakra-ui/system": "^2.6.0",
+    "@chakra-ui/tag": "^3.1.0",
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
```

Comment:
Not purely chakra related, but ensures we get the latest emotion minor.
This is only really used by the wherever plugin:

<img width="575" alt="image" src="https://github.com/shapeshift/web/assets/17035424/2cac5f6d-0791-4b3b-9b2f-cdc1ea8ad859">